projector-plugin/src/main/kotlin/SessionAction.kt

```diff
@@ -17,44 +17,32 @@
  * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  */
 import com.intellij.openapi.actionSystem.AnActionEvent
+import com.intellij.openapi.actionSystem.PlatformDataKeys
 import com.intellij.openapi.project.DumbAwareAction
-import org.jetbrains.projector.server.ProjectorServer
-import javax.swing.JOptionPane
+import com.intellij.openapi.ui.DialogWrapper
 
-class CopyUrlAction : DumbAwareAction() {
+class SessionAction : DumbAwareAction() {
 
   override fun actionPerformed(e: AnActionEvent) {
-    require(ProjectorService.instance.currentSession != null) {
+    require(ProjectorService.isSessionRunning) {
```

Comment:
```suggestion
    check(ProjectorService.isSessionRunning) {
```

Change ex to IllegalState